solr/core/src/java/org/apache/solr/core/ConfigSetService.java

```diff
@@ -54,6 +54,28 @@
 
   public static ConfigSetService createConfigSetService(CoreContainer coreContainer) {
     final ConfigSetService configSetService = instantiate(coreContainer);
+
+    // bootstrap conf if provided
```

Comment:
create is now bootstrapping three things: bootstrap_conf, bootstrap_confdir, and the `_default`.  Let's bemore consistent -- either have one method doing these, or one method for each.  In this PR it's lopsided and a try-catch around two of those which feels arbitrary.